app/wizards/schools/register_ect_wizard/registration_session.rb

```diff
@@ -0,0 +1,95 @@
+module Schools
+  module RegisterECTWizard
```

Comment:
This feels like a concern to me, which should live elsewhere, especially as there are other subclasses. All the classes in the wizard directories are steps, except for those which extend the wizard base class.  Obviously, this wizard is the chunkiest, and this class is specifically for this wizard.  But it feels like a lib or a concern, especially as it is described as "decorator for the SessionRepository".

Reply:
I see where you're coming from. There's definitely an argument for extracting it into a shared "decorator"concern. For this refactor I kept it under the wizard name space because:

- It's tightly coupled to the ect registration journey, nothing else consumes it yet, so lifting it into a shared concern feels premature

- Keeping it next to the steps makes the wiring easy to follow while we reshape things

- Once the teams comfortable with the structure (and we've mirrored it for mentors), we can revisit whether it's worth moving it into a shared concern. 

So yeah, we might end up moving it, but i'd like to land the refactor first and then decide if a shared abstraction adds value.